analysis/tc_tomcat.go

```diff
@@ -1,134 +1,748 @@
 package analysis
 
 import (
+	"github.com/konveyor/go-konveyor-tests/data"
+	"github.com/konveyor/go-konveyor-tests/hack/addon"
 	"github.com/konveyor/tackle2-hub/api"
 )
 
 var Tomcat = TC{
-	Name: "Customer Tomcat Legacy - shoud never fail",
-	Application: api.Application{
-		Name: "Customer Tomcat Legacy",
-		Repository: &api.Repository{
-			Kind: "git",
-			URL:  "https://github.com/konveyor/example-applications.git",
-			Path: "example-1",
+	Name:        "Customer Tomcat Legacy - shoud never fail",
+	Application: data.CustomerTomcatLegacy,
+	Task:        Analyze,
+	Labels: addon.Labels{
+		Included: []string{
+			"konveyor.io/target=cloud-readiness",
+			"konveyor.io/target=linux",
 		},
 	},
-	Task: Analyze,
-	Targets: []string{
-		"cloud-readiness",
-		"linux",
-	},
 	Analysis: api.Analysis{
-		Effort: 1,
+		Effort: 2,
 		Issues: []api.Issue{
 			{
-				Category:    "cloud-mandatory",
-				Description: "Trivial change or 1-1 library swap",
+				Category:    "mandatory",
+				Description: "Hardcoded IP Address",
 				Effort:      1,
-				RuleSet:     "DiscoverHardcodedIPAddressRuleProvider",
-				Rule:        "DiscoverHardcodedIPAddressRuleProvider",
+				RuleSet:     "discovery-rules",
+				Rule:        "hardcoded-ip-address",
 				Incidents: []api.Incident{
 					{
-						File:    "persistence.properties",
-						Line:    2,
-						Message: "Hard-coded IP: 169.60.225.216When migrating environments, hard-coded IP addresses may need to be modified or eliminated.",
+						File:     "/addon/source/example-applications/example-1/src/main/resources/persistence.properties",
+						Line:     2,
+						Message:  "When migrating environments, hard-coded IP addresses may need to be modified or eliminated.",
+						CodeSnip: "jdbc.url=jdbc:oracle:thin:@169.60.225.216:1521/XEPDB1",
+					},
+					{
```

Comment:
This is actually showing the other bug that we still need to fix, that the builtin provider for java needs to ignore target folder.

But for now this is correct.